FreeRTOS/Test/Target/tests/smp/template/README.md

```diff
@@ -0,0 +1,28 @@
+# How to add a new test?
```

Comment:
```
# How to add a new test?
1. Create a directory in the `FreeRTOS/Test/Target/tests` directory which will
   contain the test. For example: `FreeRTOS/Test/Target/tests/smp/multiple_tasks_running`.
1. Copy the `test_name.c` and `test_config.h` files from this template
   directory to the newly created directory above.
1. Rename the `test_name.c` according to the test name.
1. Implement the test in the above test file.
1. Add any FreeRTOS specific configuration required for the test to `test_config.h`.

# How to add a new target?
1. Create a target specific directory in the `FreeRTOS/Test/Target/boards` directory.
1. Create required build files.

# How to add a test to a target
1. Create a directory in the target's directory which will contain
   the test. For example: `FreeRTOS/Test/Target/boards/pico/tests/smp/multiple_tasks_running`.
1. Create a C file and invoke the test case from a task. The invocation
   usually looks like the following:
    ```c
    void prvTestRunnerTask( void * pvParameters )
    {
        /* Invoke the test case. */
        vRunTestCaseName();
    }
1. Add the file created above and the test case file to the build system used
   for the target.
```